Changelog.md

```diff
@@ -1,3 +1,12 @@
+# Version 15.1.1
+
+## General
+
+* Azure Kinect: Stability improvements
```

Comment:
I would prefer to list the K4A-related change using the subheading `## Azure Kinect`, like it was done for specific camera related changes in older releases.

Reply:
Reflection time expired - I just restructed it as proposed ;-)